modules/restoring-red-hat-quay-standalone.adoc

```diff
@@ -0,0 +1,218 @@
++
+[source,terminal]
+----
+\q
+----
+
+. Set the password for your PostgreSQL deployment by running the following command:
++
+[source,terminal]
+----
+$ psql "host=<user_name>.postgres.database.azure.com port=5432 dbname=example_restore_registry_quay_database user=w..y " -W <  /tmp/quay-backup/quay-backup.sql
+----
++
+Example output:
++
+----
+SET
+SET
+SET
+SET
+SET
+----
++
+If you are moving from one PostgreSQL server to another PostgreSQL server, you must update your `DB_URI` setting in your `config.yaml` file before restarting the {productname} deployment. 
+
+. In the `/opt/new-quay-install` directory, print the contents of your `DISTRIBUTED_STORAGE_CONFIG` bundle:
++
+[source,terminal]
+----
+$ cat config.yaml | grep DISTRIBUTED_STORAGE_CONFIG -A10
+----
++
+Example output:
++
+[source,yaml]
+----
+DISTRIBUTED_STORAGE_CONFIG:
+   default:
+DISTRIBUTED_STORAGE_CONFIG:
+   default:
+    - S3Storage
+    - s3_bucket: <user_name_2>
+      storage_path: /registry 
+      s3_access_key: <s3_access_key>
+      s3_secret_key: <s3_secret_key>
+      host: <host_name>
+----
++
+[NOTE]
+====
+Your `DISTRIBUTED_STORAGE_CONFIG` in `/opt/new-quay-install` must be updated before restarting your {productname} deployment. 
+====
+
+. Export the `AWS_ACCESS_KEY_ID` by using the `access_key` credential obtained in Step 13:
++
+[source,terminal]
+----
+$ export AWS_ACCESS_KEY_ID=<access_key> 
+----
+
+. Export the `AWS_SECRET_ACCESS_KEY` by using the `secret_key` obtained in Step 13:
++
+[source,terminal]
+----
+$ export AWS_SECRET_ACCESS_KEY=<secret_key>
+----
+
+. Create a new s3 bucket by entering the following command:
++
+[source,terminal]
+----
+$ aws s3 mb s3://<user_name>  --region us-east-2
+make_bucket: <user_name>
+----
++
+Example output:
++
+[source,terminal]
+----
+$ make_bucket: quay
+----
+
+. Upload all blobs to the new s3 bucket by entering the following command: 
++
+[source,terminal]
+----
+$ aws s3 sync --no-verify-ssl \
+--endpoint-url <example_endpoint_url> <1>
+/tmp/quay-backup/blob-backup/. s3://quay/
+----
+<1> The {productname} registry endpoint must be the same before backup and after restore. 
++
+Example output: 
++
+[source,terminal]
+----
+upload: ../../tmp/quay-backup/blob-backup/datastorage/registry/sha256/50/505edb46ea5d32b5cbe275eb766d960842a52ee77ac225e4dc8abb12f409a30d to s3://quay/datastorage/registry/sha256/50/505edb46ea5d32b5cbe275eb766d960842a52ee77ac225e4dc8abb12f409a30d
+upload: ../../tmp/quay-backup/blob-backup/datastorage/registry/sha256/27/27930dc06c2ee27ac6f543ba0e93640dd21eea458eac47355e8e5989dea087d0 to s3://quay/datastorage/registry/sha256/27/27930dc06c2ee27ac6f543ba0e93640dd21eea458eac47355e8e5989dea087d0
+upload: ../../tmp/quay-backup/blob-backup/datastorage/registry/sha256/8c/8c7daf5e20eee45ffe4b36761c4bb6729fb3ee60d4f588f712989939323110ec to s3://quay/datastorage/registry/sha256/8c/8c7daf5e20eee45ffe4b36761c4bb6729fb3ee60d4f588f712989939323110ec
+...
+----
+
```

Comment:
Here,  it is better to remind customer to update storage setting in the `config.yaml` before restarting the quay like line 139. 
```
DISTRIBUTED_STORAGE_CONFIG:
   default:
DISTRIBUTED_STORAGE_CONFIG:
   default:
    - S3Storage
    - s3_bucket: <new_bucket_name>       
      storage_path: /registry 
      s3_access_key: <s3_access_key>
      s3_secret_key: <s3_secret_key>
      host: <host_name>
```